lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -163,6 +163,7 @@ define([
                 // Perform a search
                 teamsTabView.$('.search-field').val('foo');
                 teamsTabView.$('.action-search').click();
+                Backbone.history.navigate('topics/' + TeamSpecHelpers.testTopicID + '/search', {trigger: false});
```

Comment:
Note: this is a bit of a hack. In Jasmine test, the Backbone history usually isn't what it would be in the real app. This line just sets the URL fragment without triggering any callbacks, which allows the changes in `teams_tab.js` to correctly detect the fragment.